packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -285,3 +316,14 @@ export interface UTXO {
   txid: string
   vout: number
 }
+
+export interface DepositVault {
+  vaultId: string
+  from: string
+  amount: string // amount@symbol
+}
+
+export interface TakeLoanMetadata {
+  vaultId: string
+  amounts: string // amount@symbol
```

Comment:
```suggestion
  amount: string // amount@symbol
```

@surangap this should probably be standardized on c++ side

Reply:
Looking at the C++ side sources, `amounts` suites this better since it can accept an `amount@symbol` array.
Refer https://github.com/DeFiCh/ain/blob/f3add1d49a4e1e8ee4f2ec6d5f572a07b232248d/src/masternodes/rpc_loan.cpp#L1042
https://github.com/DeFiCh/ain/blob/f3add1d49a4e1e8ee4f2ec6d5f572a07b232248d/src/rpc/rawtransaction_util.cpp#L107
@canonbrother we should add a test to cover that.

Reply:
actually we need to change the interface

Reply:
sure will update test

Reply:
https://github.com/DeFiCh/jellyfish/pull/699/files#r719062793